.pre-commit-config.yaml

```diff
@@ -1,45 +1,81 @@
+exclude: ^\.pixi
 repos:
-  - repo: https://github.com/Quantco/pre-commit-mirrors-insert-license
-    rev: 1.3.0
+  - repo: local
     hooks:
+      # ensure pixi environments are up to date
+      # workaround for https://github.com/prefix-dev/pixi/issues/1482
+      - id: pixi-install
+        name: pixi-install
+        entry: pixi install -e default -e lint
+        language: system
+        always_run: true
+        require_serial: true
+        pass_filenames: false
       - id: insert-license
```

Comment:
i don't think we need this in open source repos

Reply:
My reading of the situation was that we do in order to insert the following header:
```
# # Copyright (c) QuantCo 2024-2024
# # SPDX-License-Identifier: BSD-3-Clause
```
https://github.com/Quantco/metalearners/blob/main/metalearners/metalearner.py#L1-L2

Reply:
the extra `# ` is definitely too much. not sure whether we need the license header in open-source repos

Reply:
We should have that, too. It is not as important as for internal things, but we should keep it here.